tests/e2e/instrumentation-nodejs/01-install-app.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-with-sidecar
+spec:
+  selector:
+    matchLabels:
+      app: my-pod-with-sidecar
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: my-pod-with-sidecar
+      annotations:
+        sidecar.opentelemetry.io/inject: "true"
+        instrumentation.opentelemetry.io/inject-nodejs: "true"
+    spec:
+      containers:
+      - name: myapp
+        image: ghcr.io/anuraaga/express-hello-world:latest
```

Comment:
We should probably create an application that would call an API on our test apps (java, js) that would result in a trace. 
Then report the trace to an inmemory storage and asser it exists (e.g. what we do in the java auto-instrumentation tests).

Reply:
Yup think we will want this